### PR TITLE
[explorer] - add next epoch gas to validators table

### PR DESCRIPTION
**Pull request by mamos-mysten**
## Description 

Adds `nextEpochGasPrice` to Validators Table / Details and makes a few small style changes to match designs.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Comment by mamos-mysten** [comment]
Current dependencies on/for this PR:
* main
  * **PR #9490** <a href="https://app.graphite.dev/github/pr/MystenLabs/sui/9490" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/MystenLabs/sui/9490?utm_source=stack-comment).

File: apps/explorer/src/pages/validators/Validators.tsx [hunk 1]
```diff
@@ -44,14 +44,15 @@ export function validatorsTableData(
...
                 validator.suiAddress
             );
-
             return {
                 name: {
                     name: validatorName,
                     logo: validator.imageUrl,
                 },
                 stake: totalStake,
+                nextEpochStake: validator.nextEpochStake,
```
Comment (mamos-mysten) [inline apps/explorer/src/pages/validators/Validators.tsx:53]: also added this, as it seemed useful

File: apps/explorer/src/components/validator/ValidatorStats.tsx [hunk 2]
```diff
@@ -161,6 +161,15 @@ export function ValidatorStats({
                                     tooltip="Latest Narwhal round for this epoch."
                                     unavailable
                                 />
+                                <Stats
+                                    label="Next Epoch Gas Price"
+                                    tooltip="Estimated gas fees in the upcoming epoch."
```
Comment (Jordan-Mysten) [inline apps/explorer/src/components/validator/ValidatorStats.tsx:166]: This sounds wrong, this should be their vote for the gas price in the next epoch, right?

**Review by Jordan-Mysten — APPROVED** [review]
One nit that you'll want to follow-up on, but looks good overall.

File: apps/explorer/src/components/validator/ValidatorStats.tsx (hunk 2 above)
Comment (mamos-mysten) [reply apps/explorer/src/components/validator/ValidatorStats.tsx]: updated the wording a bit - but I pulled this from here: https://github.com/MystenLabs/sui/blob/f389b8e83e7e408039992a577dfb723d13ba1675/crates/sui-framework/sources/governance/validator.move#L134

**Review by Jordan-Mysten — APPROVED** [review]

**Pull request merged**